lib/generators/apple_auth/apple_sign_in_controller/apple_sign_in_controller_generator.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module AppleAuth
+  module Generators
+    class AppleSignInControllerGenerator < Rails::Generators::Base
+      source_root File.expand_path('templates', __dir__)
+      argument :scope, required: false, default: ''
+
+      def copy_apple_sign_in_controller_file
+        @scope_prefix = scope.blank? ? '' : scope.camelize
```

Comment:
as we have:
```
argument :scope, required: false, default: ''
```

wouldn't the default value of scope be `''`?

If that's the case, then we can write this as `@scope_prefix = scope.camelize`

Reply:
In that case, when scope eq '' will raise an exception, we cant apply camelize method to an empty string